### PR TITLE
feat: added AuthenticateUser() and GetUserLicenses()

### PR DESCRIPTION
**Pull request by ahmad-kemsan**

### CODE REVIEW

File: src/main/java/com/cryptlex/lexactivator/LexActivatorException.java [hunk 1]
```diff
@@ -207,6 +207,10 @@ public static String getErrorMessage(int errorCode) {
             message = "Release channel not set.";
             break;
 
+        case LA_E_USER_NOT_AUTHENTICATED:
+            message = "Permission denied.";
```
Comment (adnan-kamili) [inline src/main/java/com/cryptlex/lexactivator/LexActivatorException.java:211]: fix message

File: src/main/java/com/cryptlex/lexactivator/LexActivatorException.java [hunk 2]
```diff
@@ -566,6 +570,13 @@ public static String getErrorMessage(int errorCode) {
      * MESSAGE: Release channel not set.
      */
     public static final int LA_E_RELEASE_CHANNEL = 86;
+    
+    /*
+     * CODE: LA_E_USER_NOT_AUTHENTICATED
+     *
+     * MESSAGE: Permission denied.
```
Comment (adnan-kamili) [inline src/main/java/com/cryptlex/lexactivator/LexActivatorException.java:577]: fix message

File: src/main/java/com/cryptlex/lexactivator/LexActivatorException.java (hunk 1 above)
Comment (ahmad-kemsan) [reply src/main/java/com/cryptlex/lexactivator/LexActivatorException.java]: Fixed.

File: src/main/java/com/cryptlex/lexactivator/LexActivatorException.java (hunk 2 above)
Comment (ahmad-kemsan) [reply src/main/java/com/cryptlex/lexactivator/LexActivatorException.java]: Fixed.

**Review by adnan-kamili — APPROVED** [review]

**Pull request merged**